client/src/Components/Footer/Footer.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./Footer.scss";
+
+const Footer = () => {
+  return (
+    <footer>
+      <p>Copyright © 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd</p>
+    </footer>
```

Comment:
text must come from json, config or constant file

Reply:
fixed and pushed the changes